packit_service/worker/handlers/github_handlers.py

```diff
@@ -179,11 +197,28 @@ def run(self) -> TaskResults:
                 " to the issue comment.\n"
             )
 
-            self.project.create_issue(
+            issue = self.project.create_issue(
                 title=f"[packit] Propose update failed for release {self.data.tag_name}",
                 body=body_msg,
             )
 
+            if any(FILE_DOWNLOAD_FAILURE in error for error in errors.values()):
+                retry_message = (
+                    "Since your archive was not found in PyPI,"
+                    "we will try in 10 minutes again."
```

Comment:
It's not only about `PyPI`.

```suggestion
                    "Since your archive was not found, we can't do the update in Fedora."
                    "We will try in 10 minutes again."
```